www/default.js

```diff
@@ -130,6 +130,7 @@ function change_cost(duration) {
   });
 
   var hour_multipliers = {
+    "secondly": 1 / (60 * 60),
```

Comment:
Why not simply 1 / 3600 ?

Reply:
well, I decided to follow the pattern:
`"weekly": (7 * 24),`

Reply:
Okay, for consistency LGTM

Reply:
Okay, for consistency LGTM